Fix modal verb "must" (#4222)
cplusplus/draft: source/grammar.tex

In particular, the grammar described here accepts
a superset of valid \Cpp{} constructs.
Disambiguation rules~(\ref{stmt.ambig}, \ref{dcl.spec}, \ref{class.member.lookup})

> shall be applied to distinguish expressions from declarations.
> Further, access control, ambiguity, and type rules shall be used

Reviewer: ```suggestion
are applied to distinguish expressions from declarations.
Further, access control, ambiguity, and type rules are used
```

We can't "shall" from an informative Annex, and I think this is supposed to be descriptive rather than introducing a requirement.

Author: done
